rocky/katalogus/templates/normalizer_detail.html

```diff
@@ -62,7 +64,14 @@ <h2>{% translate "Produces" %}</h2>
                     {% endblocktranslate %}
                 </p>
                 <p>
-                    {% for ooi in plugin.produces %}{{ ooi }}{% endfor %}
+                    {% if plugin.produces %}
+                        {% for ooi_type in plugin.produces %}
+                            <a href="{% url "ooi_list" organization_code=organization.code %}?ooi_type={{ ooi_type|urlencode:"" }}">{{ ooi_type }}</a>
+                            {% if not forloop.last %}-{% endif %}
+                        {% endfor %}
+                    {% else %}
+                        No known produces OOI's listed. N.B. This list might not be complete.
```

Comment:
```suggestion
                        No known produces OOIs listed. N.B. This list might not be complete.
```

Reply:
"No known produces OOIs listed." was a bit hard to understand, but I don't have an alternative (yet)